trillian_map_api.proto

```diff
@@ -93,6 +93,11 @@ message SetMapLeavesRequest {
   // to continue mapping from an external data source.
   reserved 4;
   bytes metadata = 5;
+  // The map revision to associate the leaves with. The request will fail if
+  // this revision already exists, does not match the current write revision, or
```

Comment:
Hm, probably suffices to remove all except "does not match the current write revision", as the others follow from it.